Sources/JNI/JNIObjects.swift

```diff
@@ -87,6 +87,10 @@ open class JNIObject {
         return try jni.GetField(fieldName, from: self.instance)
     }
 
+    public func getField(_ fieldName: String, fieldJavaClassName: String) throws -> JavaObject {
+        return try jni.GetField(fieldName, fieldJavaClassName: fieldJavaClassName, from: self.instance)
```

Comment:
Cool looks good, ideally we'd add getters for primitive fields too (Int etc) but we can also do that later when/if we need it

Reply:
I think what you mean is already covered by the generic methods above.